fontbe/src/orchestration.rs

```diff
@@ -210,12 +213,26 @@ where
     T::read(FontData::new(&buf)).unwrap()
 }
 
-fn raw_hmtx_from_file(file: &Path) -> RawHmtx {
+pub struct Bytes {
```

Comment:
you have an identical `Raw` struct in fontbe::hmtx module...

Reply:
I have 2, need 1, and would prefer ot have 0 rofl. Will fix.

Reply:
Keeping this one.